06_create_cluster.sh

```diff
@@ -34,6 +34,12 @@ if [[ ! -z "${ENABLE_LOCAL_REGISTRY}" ]]; then
     
     oc patch configs.samples cluster \
       --type merge --patch "{\"spec\":{\"samplesRegistry\":\"${LOCAL_REGISTRY_DNS_NAME}\"}}"
+    
+    # Some CI tests try to test scale in and out of machines. For that purpose,
+    # if NUM_EXTRA_WORKERS is set, we enable extra worker for that test.
+    if [ -f "${OCP_DIR}/extra_host_manifests.yaml" ]; then
+	    oc apply -f "${OCP_DIR}/extra_host_manifests.yaml"
+    fi
```

Comment:
A wait condition to verify that all the hosts were setup could be useful, and it allow to detect issues earlier. In addition, there wouldn't be the risk to run the tests while the hosts are coming up

Reply:
Good point. Because test failed when bmh is not in ready state.